tools/go.mod

```diff
@@ -6,12 +6,17 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/buildpacks/imgutil v0.0.0-20200831154319-afd98bd2f655
 	github.com/buildpacks/lifecycle v0.8.1
-	github.com/docker/docker v1.4.2-0.20200221181110-62bd5a33f707
+	github.com/docker/docker v1.14.0-0.20190319215453-e7b5f7dbe98c
 	github.com/golang/mock v1.4.4
 	github.com/golangci/golangci-lint v1.30.0
-	github.com/google/go-containerregistry v0.0.0-20200313165449-955bf358a3d8
+	github.com/google/go-containerregistry v0.1.2-0.20200804170047-b0d31a182cf0
```

Comment:
To avoid dependency hell we should test early to see if this code has problems compiling with ggcr `v0.1.4` and the associated version of `docker` -> https://github.com/buildpacks/imgutil/issues/70